lib/mongo_ecto.ex

```diff
@@ -758,7 +758,13 @@ defmodule Mongo.Ecto do
   @list_collections_query ["$and": [[name: ["$not": special_regex]],
                                     [name: ["$not": migration_regex]]]]
 
-  defp list_collections(repo, opts) do
+  def list_collections(_ = "3.2", repo, opts) do
+    colls = command(repo, %{"listCollections": 1}, opts)
+
+    colls["cursor"]["firstBatch"] |> Enum.map(&Map.fetch!(&1, "name"))
+  end
```

Comment:
`listCollections` is available since 3.0, we should use it since then. Let's only check the version is `> 3` - this should be enough and prepare us for next versions.